boilerplate/openshift/golang-osd-cluster-operator/standard.mk

```diff
@@ -110,7 +110,14 @@ envtest: isclean
 	@eval $$($(MAKE) env --no-print-directory) || (echo 'Unable to evaulate output of `make env`.  This breaks osd-operators-registry.' >&2 && exit 1)
 
 .PHONY: test
-test: envtest gotest yaml-validate
+test: envtest gotest yaml-validate check_generation
+
+.PHONY: check_generation
+check_generation: 
+	@$(MAKE) -s isclean --no-print-directory || (echo 'check_generation requires a clean project to run. Please commit or revert you changes first.' >&2 && exit 1) 
+	@$(MAKE) -s gogenerate --no-print-directory
```

Comment:
As far as I remember, there is no ordering in the deps as they can be ran in parallel, meaning we can't have `gogenerate` as a deps. And because of that, I thought it would be more consistent to have all done the same way and with appropriate error message

Reply:
> As far as I remember, there is no ordering in the deps as they can be ran in parallel

Right you are. Withdrawn.